### PR TITLE
More detailed README on setup, tests and deployment

### PR DESCRIPTION
**Pull request by brenoferreira**
Part of #173 

Includes a more detailed documentation on how to setup the project, run tests and deploy.
## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,42 @@
 # cccatalog-frontend
 
-> Creative Commons Search
+## About
 
-## Build Setup
+Repository containing the [CC Search](https://ccsearch.creativecommons.org/) frontend code.
+
+CC Search aims to private the front door to a comprehensive commons of CC and public domain materials, in order to maximize not only discovery, but use and engagement with all of that content.
```
Comment (kgodey) [inline README.md]: I think "private" should be "provide"

Comment (kgodey) [reply README.md]: You may want to rephrase this to be simpler because this sounds a bit too marketing-y for a code repo. Maybe something like "CC Search is an interface to search for content that is licensed under Creative Commons licenses or in the public domain. "

**Comment by aldenstpage** [comment]
It would also be helpful to document how to configure the API URL
